Cargo.toml

```diff
@@ -45,17 +45,6 @@ exclude = [
 # Even specifying `edition = "2021"` here will not fix this
 resolver = "2"
 
-[profile.dev]
-opt-level = 0
-lto = true
```

Comment:
FYI: @jcape having lto on was causing difficulties debugging test runs.  
This looks to go all the way back to https://github.com/mobilecoinfoundation/sgx/commit/40838d72c1f0e477f36bfb4703793ce404190bcf

Reply:
I did not, I think for `dev` we should avoid any kind of optimizations to make it easier to debug. 
It could just be me and my history with c/c++, but I think most people would expect step over and step in to do it in debug/dev builds.  Only if someone did a release with debug symbols or similar do they expect to skip over code when degbugging.

Looking at [thin](https://clang.llvm.org/docs/ThinLTO.html) it seems that this is how much LTO happens and also provides an incremental linking capability, like was added in MSVC2019.

Reply:
Yeah I'm more worried about the release profile - the default `--release` doesn't include LTO at all.

Reply:
Added lto back into the release profile